src/app/shared/auth/auth.service.ts

```diff
@@ -31,7 +38,11 @@ export class AuthService {
   }
 
   getUserInfo(): User | object {
-    return JSON.parse(this.storageService.get('user'));
+    return this.http.post('http://localhost:3004/auth/userinfo/', {token: this.storageService.get('token') || ''}, httpOptions);
```

Comment:
'token' string can be a value of `enum` and this must be env-prefixed to avoid configuration re-writes  on different envs.

Reply:
moreover it will allow you to not Repeat Yourself.

Reply:
don't understand :(. I get the token from the backend

Reply:
sorry for that, my bad